pkg/cmd/admin/top/printer.go

```diff
@@ -31,7 +31,11 @@ func printHeader(out io.Writer, columns []string) {
 }
 
 func printArray(out io.Writer, values []string) {
-	printValue(out, strings.Join(values, ","))
+	if len(values) == 0 {
+		printValue(out, "<none>")
+	} else {
+		printValue(out, strings.Join(values, ", ")+" ")
```

Comment:
It used to be a placeholder when values where empty, but I've decided to go with `<none>` instead, will remove.